src/mongocrypt-ctx-private.h

```diff
@@ -54,6 +63,7 @@ typedef struct __mongocrypt_ctx_opts_t {
    _mongocrypt_key_alt_name_t *key_alt_names;
    mongocrypt_encryption_algorithm_t algorithm;
    _mongocrypt_ctx_opts_azure_kek_t azure_kek;
+   _mongocrypt_ctx_opts_gcp_kek_t gcp_kek;
```

Comment:
We should use a union here for the azure and gcp options. You could a file a bug to do the same for AWS in the future.

Reply:
Good point, I noted in https://jira.mongodb.org/browse/MONGOCRYPT-283 to do this for AWS.